packages/backend/sendvote/app.js

```diff
@@ -158,6 +158,14 @@ exports.handler = async event => {
   const dbKey = `${nounId}||${blockhash}`;
   const endpoint = `${context.domainName}/${context.stage}`;
 
+  // Retrieve connected users
+  const connections = await retrieveConnections();
+  const thisConnection = connections.find(i => i.connectionId === context.connectionId);
```

Comment:
Can you update this to align with the latest `main` fix? Apologies, this is different than how it used to work earlier but should be slightly more robust.

Reply:
I have updated this, and it now goes after the time buffer check. It can't go after the vote save call since if the user hasn't passed the captcha, he should not be able to send(save) votes.